scripts/get-language-providers.sh

```diff
@@ -24,7 +24,7 @@ download_release() {
 }
 
 # shellcheck disable=SC2043
-for i in "github.com/pulumi/pulumi-java java v1.1.0" "github.com/pulumi/pulumi-yaml yaml v1.13.0" "github.com/pulumi/pulumi-dotnet dotnet v3.72.0"; do
+for i in "github.com/pulumi/pulumi-java java v1.1.0" "github.com/pulumi/pulumi-yaml yaml v1.13.1" "github.com/pulumi/pulumi-dotnet dotnet v3.73.0"; do
```

Comment:
I think you need to also bump https://github.com/pulumi/pulumi/blob/9e7d51b1a5f6913d15d8635c8285f8122cf6e0b9/pkg/codegen/testing/test/helpers.go#L381

Reply:
What a catch -- thanks so much :pray:

Reply:
Is it worth a comment right above this line to remind us about this?

Reply:
Excellent shout -- done :+1:

Reply:
It's also in https://github.com/pulumi/pulumi-dotnet/blob/main/CONTRIBUTING.md, but good to have here as well, since people are likely to jump straight into this file.